apps/web/src/db/schema/auth/users.ts

```diff
@@ -32,18 +29,18 @@ export const kindleCountriesEnum = readfort.enum('kindle', [
 export const users = readfort.table(
   'users',
   {
-    id: text('id')
-      .primaryKey()
-      .$defaultFn(() => crypto.randomUUID()),
+    /* id: text('id') */
+    /*   .primaryKey() */
+    /*   .$defaultFn(() => crypto.randomUUID()), */
+    id: bigserial('id', { mode: 'number' }).notNull().primaryKey(),
```

Comment:
This requires changing next-auth types so it can understand the the ID is not a text but a bigserial 

This is how Cal.com does it
https://github.com/calcom/cal.com/blob/c5483c81decfe9640145e0ea8177647d394ad321/packages/types/next-auth.d.ts#L13